move install script to the top (#10490)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/linux-install-manually.md

> [!WARNING]
> Upgrading your operating system to a new major version after the product installation requires the product to be reinstalled. You need to [Uninstall](linux-resources.md#uninstall-defender-for-endpoint-on-linux) the existing Defender for Endpoint on Linux, upgrade the operating system, and then reconfigure Defender for Endpoint on Linux following the below steps.

## Installer script

The easiest way to deploy Defender for Endpoint on Linux is with an automated [installer bash script](https://github.com/microsoft/mdatp-xplat/blob/master/linux/installation/mde_installer.sh) provided in our [public GitHub repository](https://github.com/microsoft/mdatp-xplat/).

> The script identifies the distribution and version, simplifies the selection of the right repository, sets up the device to pull the latest package, and combines the product installation and onboarding steps. Details on the steps performed can be found in the manual set up instructions below this section.

Reviewer: ```suggestion
The script identifies the distribution and version, simplifies the selection of the right repository, sets up the device to pull the latest package, and combines the product installation and onboarding steps. Details on the steps to be performed are provided in the manual setup instructions following this section.
```
